pkgs/by-name/im/immersed/linux.nix

```diff
@@ -2,8 +2,63 @@
 , version
 , src
 , meta
+, pkgs
+, lib
 , appimageTools
+, makeWrapper
+, libgpg-error
+, fontconfig
+, libSM
+, libGL
+, libz
+, mesa
+, wayland
+, pipewire
+, fribidi
+, harfbuzz
+, freetype
+, libthai
+, e2fsprogs
+, zlib
+, libp11
+, xorg
 }:
-appimageTools.wrapType2 {
-  inherit pname version src meta;
+
+let
+  xorgLibs = with xorg; [
+    libX11
+  ];
+  src' = appimageTools.extract {
+    inherit pname version;
+    src = src;
+
+    postExtract = ''
+      # Because of https://github.com/NixOS/nixpkgs/issues/267408
+      cp ${libgpg-error}/lib/* $out/usr/lib/
+    '';
```

Comment:
```suggestion
    # Because of https://github.com/NixOS/nixpkgs/issues/267408
    postExtract = ''
      cp ${libgpg-error}/lib/* $out/usr/lib/
    '';
```

Reply:
resolved by: 172d66c38e22c0e56a2a9362963bdc692601eb1d